### DISCUSSION TITLE
Copilot for Eclipse and Dbeaver

### DISCUSSION THREAD

**Discussion by ghost** [community/community#17859] [Copilot Conversations]
Is there any possibility of making a plugin of GitHub copilot for eclipse ide that will also work with Dbeaver just like vrapper.

**Comment by dstango** [comment]
I'd be interested in a plugin for Eclipse, too!

**Comment by alejandromh91** [comment]
masecla22 try to do it using the IntelliJ plugin, but i dont know if he made any progress... 

the repo: https://github.com/masecla22/eclipse-github-copilot-integration

**Comment by brianvfernandes** [comment]
We built a plugin for Eclipse 2023-06 and up - [Copilot4Eclipse](https://marketplace.eclipse.org/content/copilot4eclipse). Works just fine with the DBeaver SQL editor for instance.
